src/cmd/sync.rs

```diff
@@ -27,7 +27,12 @@ use tracing::warn;
 pub struct Args {
     /// target cros repo dir. If omitted, current directory will be used.
     #[argh(option)]
-    repo: Option<String>,
+    cros: Option<String>,
+
+    /// target android repo dir. --version will be tm or rvc. Only one of --cros
```

Comment:
nit: "--version can be tm or rvc" (since this explanation is not about the future but explaining about the restriction on the option)

Also, are we restricting --version value to tm or rvc? If not, may be you can rephrase to:
"target android repo dir. When this flag is specified, --version option can be the branch name to sync to."